### DISCUSSION TITLE
Standalone Package

### DISCUSSION THREAD

**Discussion by waynelapierre** [posit-dev/positron#4660] [Q&A]
I prefer VSCode over Poistron. Is there a VSCode package that offers the R features of Positron?

**Comment by jmcphers** [comment]
No, there can't be -- one of the reasons we made Positron a new IDE (instead of a VS Code extension) is that a really first-class R experience requires changes to the IDE that are beyond what a VS Code extension can do.

There is a community-built R extension for VS Code that might do what you want, depending on your needs. Here's a writeup: https://code.visualstudio.com/docs/languages/r
